### ISSUE TITLE
Basic HTML notice's layout looks broken

### ISSUE THREAD

**Issue by EricDunsworth** [gc-da11yn/wcag-issues#3]
I tried [Googling "wcag failure for not using list" (without quotes)](https://www.google.com/search?q=wcag+failure+for+not+using+list) yesterday and one of the first page's search results linked to the [common found WCAG 2.0 issues page... in basic HTML mode](https://wet-boew.github.io/GCWeb/index-en.html?wbdisable=true) :D.

Here's a screenshot of what its basic HTML notice looked like:
![basic-html-notice_a11y-canada-ca](https://github.com/user-attachments/assets/3afc28b6-d0dd-4291-b908-2858f4cd0a50)

Here's what it's supposed to look like (taken from [GCWeb's home page in basic HTML mode](https://wet-boew.github.io/GCWeb/index-en.html?wbdisable=true)):
![basic-html-notice_gcweb](https://github.com/user-attachments/assets/e634ec0c-62fc-4f16-a9ba-973d1900ee14)

I think what's happening is that the WCAG 2.0 issues section is using buggy outdated versions of WET/GCWeb.

The same problem was previously reported in wet-boew/GCWeb#1781 + wet-boew/GCWeb#1868 in 2021 and subsequently fixed.

**More info:**
* The WCAG 2.0 issues repo is currently using...
  * **WET v4.0.43.1**
    * https://github.com/gc-da11yn/wcag-issues/blob/bb2f784cced13762ea9c2cf7cf3319ee17091ded/package.json#L41-L44
    * https://github.com/gc-da11yn/wcag-issues/blob/bb2f784cced13762ea9c2cf7cf3319ee17091ded/package-lock.json#L10-L13
    * https://github.com/gc-da11yn/wcag-issues/blob/bb2f784cced13762ea9c2cf7cf3319ee17091ded/package-lock.json#L5029-L5033
    * https://github.com/gc-da11yn/wcag-issues/blob/bb2f784cced13762ea9c2cf7cf3319ee17091ded/package-lock.json#L9240-L9243
  * **GCWeb v9.4.1**
    * https://github.com/gc-da11yn/wcag-issues/blob/bb2f784cced13762ea9c2cf7cf3319ee17091ded/package.json#L41-L44
    * https://github.com/gc-da11yn/wcag-issues/blob/bb2f784cced13762ea9c2cf7cf3319ee17091ded/package-lock.json#L10-L13
    * https://github.com/gc-da11yn/wcag-issues/blob/bb2f784cced13762ea9c2cf7cf3319ee17091ded/package-lock.json#L1588-L1593
    * https://github.com/gc-da11yn/wcag-issues/blob/bb2f784cced13762ea9c2cf7cf3319ee17091ded/package-lock.json#L6497-L6500
* As for where GCWeb itself stands...
  * It's currently at [v15.5.0](https://github.com/wet-boew/GCWeb/releases/tag/v15.5.0)
  * WET dependency is currently at [v4.0.81.1](https://github.com/wet-boew/wet-boew/releases/tag/v4.0.81.1)

Should be possible to fix the issue by upgrading to newer versions of WET/GCWeb. But keep in mind that there might be breaking changes that'd need to be accounted for in newer versions of those dependencies.

Thanks :)!

**Comment by shawnthompson** [comment]
Fixed, not sure if we should keep this site online but thanks for flagging it.
